### DISCUSSION TITLE
Chapters support in YouTube Music

### DISCUSSION THREAD

**Discussion by MulverineX** [ReVanced/revanced-discussions#165] [Question]
Since clearly YouTube doesn't care about implementing the feature, it would be lovely for it to be added by ReVanced. Huge bonus points if along the road you're able to optionally queue songs in between chapters and queue individual chapters, just like on Apple Music.

**Comment by ArjixWasTaken** [comment]
why do you need chapters in yt music? is it for podcasts or smth?

**Reply by MulverineX** [reply]
It's mainly for uploaded live sets, single-video album uploads, and mixes. Lots of those videos have chapters and/or will have them through SponsorBlock. On Apple Music uploaded sets/mixes contain discrete tracks, and it would be cool if Vanced YTM could simulate this, queuing up a song in between is nice, and playing just one of the songs is great for special live versions that don't have a quality discrete upload. Or at the very least display what song (chapter) in the mix is playing & allow selection of others during playback like the base app.

**Comment by grester** [comment]
This would be a great addition to revanced indeed.
